### DISCUSSION TITLE
TailwindCSS production mode is not including any decimal classes like py-2.5, py-3.5

### DISCUSSION THREAD

**Discussion by jinalptl1993** [tailwindlabs/tailwindcss#9402] [Help]
**What version of Tailwind CSS are you using?**

v2.2.19

**What build tool (or framework if it abstracts the build tool) are you using?**

postcss 8.4.16, Next.js 12.1.6, webpack-cli 4.9.2

**What version of Node.js are you using?**

v14.17.3

**What browser are you using?**

Chrome

**What operating system are you using?**

Windows

**Describe your issue**

Here is my tailwind configuration file 
```
module.exports = {
  purge: [
    "./src/pages/*.{js,jsx,ts,tsx}",
    "./src/pages/**/*.{js,jsx,ts,tsx}",
    "./src/components/**/*.{js,jsx,ts,tsx}",
    "./src/components/**/**/**/*.{js,jsx,ts,tsx}",
  ],
```

I am facing one weird issue in the PRODUCTION environment only. When I am using any classes with decimal value like 2.5, 3.5, etc It is not included in my build and so styling is not applied. 
These classes are working fine in my local environment.

Few points to note,
* I have tried to add these classes to the safelist as below.
```
safelist: ["py-2.5", "px-2.5"],
```
* I also tried the below one.
```
options: {
   safelist: ["py-2.5", "px-2.5"],
},
```
* I have tried the whitelist option as below.
```
whitelist: ["py-2.5", "px-2.5"],
```
* None of the above solutions is working for me. Any other class which I have added in any files of the above purge path is working fine. I am only having issues with decimal point classes. 

Could you please suggest to me If I am missing something here or any solution for this issue?

Thanks a lot in advance!

**Comment by adamwathan** [comment]
Hey! Please share a reproduction otherwise no real way to help, this definitely works in my own projects and in Tailwind Play so need to see a real project where it’s not working to really trace down the issue.

**Reply by jinalptl1993** [reply]
Thanks @adamwathan for your response. 
I am going to create a sample project where I can reproduce the issue and will add a link here once done.

Thanks again for your quick response.
